src/promptflow/tests/executor/e2etests/test_executor_happypath.py

```diff
@@ -233,17 +236,24 @@ def test_executor_exec_line(self, flow_folder, dev_connections):
             ("connection_as_input", "conn_node", None, None),
             ("simple_aggregation", "accuracy", {"text": "A"}, {"passthrough": "B"}),
             ("script_with_import", "node1", {"text": "text"}, None),
+            ("python_tool_with_multiple_image_nodes", "python_node_2", {"logo_content": "Microsoft and four squares"},
+             {"python_node": {"image": "logo.jpg", "image_name": "Microsoft's logo", "image_list": ["logo.jpg"]}}),
+            ("python_tool_with_multiple_image_nodes", "python_node", {
+             "image": "logo.jpg", "image_name": "Microsoft's logo"}, {},)
         ],
     )
     def test_executor_exec_node(self, flow_folder, node_name, flow_inputs, dependency_nodes_outputs, dev_connections):
         self.skip_serp(flow_folder, dev_connections)
         yaml_file = get_yaml_file(flow_folder)
+        working_dir = get_yaml_working_dir(flow_folder)
+        os.chdir(working_dir)
         run_info = FlowExecutor.load_and_exec_node(
             yaml_file,
             node_name,
             flow_inputs=flow_inputs,
             dependency_nodes_outputs=dependency_nodes_outputs,
             connections=dev_connections,
+            output_dir=("./temp"),
```

Comment:
Also add assert for node output path?
Since we made the test case more complex now.

Reply:
Added

Reply:
Added